src/error.rs

```diff
@@ -232,7 +234,7 @@ pub enum Error {
     TimeNotSynced(u32, u32),
     /// An indexer API request contains a query parameter not supported by the endpoint.
     #[error("An indexer API request contains a query parameter not supported by the endpoint.")]
-    UnsupportedQueryParameter,
+    UnsupportedQueryParameter(QueryParameter),
```

Comment:
I think for thiserror it should also be added in the message above with `{0}`

Reply:
Done